### ISSUE TITLE
Ubuntu 14.04 steam big picture mode segfaults

### ISSUE THREAD

**Issue by jhall39** [ValveSoftware/steam-for-linux#4440]
Only big picture mode,  desktop mode works fine. 
.steam/bin/panorama/etc/fonts/conf.d/41-repl-os-win.conf complains about the <test> tag containing two entries on lines 148 and 160. That's the only clue I have. Worked fine before I updated steam. 
I can't fix it,  steam replaces the file every time.

**Comment by jhall39** [comment]
Any way to roll back one or two builds?

**Comment by jhall39** [comment]
No change with latest steam client beta from May 10 2016 23:34:44

**Comment by jhall39** [comment]
rm: cannot remove \u2018/home/jhall/.steam/bin\u2019: Is a directory
Running Steam on ubuntu 14.04 64-bit
STEAM_RUNTIME is enabled automatically
Installing breakpad exception handler for appid(steam)/version(1462929557)
Installing breakpad exception handler for appid(steam)/version(1462929557)
Installing breakpad exception handler for appid(steam)/version(1462929557)
Installing breakpad exception handler for appid(steam)/version(1462929557)
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 70: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 70: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 78: saw unknown, expected number
[0511/174551:ERROR:main_delegate.cc(779)] Could not load cef_extensions.pak
[0511/174551:ERROR:browser_main_loop.cc(217)] Running without the SUID sandbox! See https://chromium.googlesource.com/chromium/src/+/master/docs/linux_suid_sandbox_development.md for more information on developing with the sandbox on.
Installing breakpad exception handler for appid(steamwebhelper)/version(20160510233344)
Installing breakpad exception handler for appid(steamwebhelper)/version(1462923224)
[0511/174551:ERROR:main_delegate.cc(779)] Could not load cef_extensions.pak
Installing breakpad exception handler for appid(steamwebhelper)/version(20160510233344)
Installing breakpad exception handler for appid(steamwebhelper)/version(1462929557)
Installing breakpad exception handler for appid(steamwebhelper)/version(1462929557)
Installing breakpad exception handler for appid(steam)/version(1462929557)
Installing breakpad exception handler for appid(steam)/version(1462929557)
Installing breakpad exception handler for appid(steam)/version(1462929557)
Installing breakpad exception handler for appid(steam)/version(1462929557)
Installing breakpad exception handler for appid(steam)/version(1462929557)
Installing breakpad exception handler for appid(steam)/version(1462929557)
Created shared memory when not owner SteamController_Shared_mem
Installing breakpad exception handler for appid(steam)/version(1462929557)
Installing breakpad exception handler for appid(steam)/version(1462929557)
PulseAudio connect failed (used only for Mic Volume Control) with error: Access denied
Installing breakpad exception handler for appid(steam)/version(1462929557)
Fontconfig warning: "/home/jhall/.local/share/Steam/bin/panorama/etc/fonts/conf.d/41-repl-os-win.conf", line 148: Having multiple values in <test> isn't supported and may not work as expected
Fontconfig warning: "/home/jhall/.local/share/Steam/bin/panorama/etc/fonts/conf.d/41-repl-os-win.conf", line 160: Having multiple values in <test> isn't supported and may not work as expected
assert_20160511174550_1.dmp[15979]: Uploading dump (out-of-process)
/tmp/dumps/assert_20160511174550_1.dmp
/home/jhall/.local/share/Steam/steam.sh: line 753: 15919 Segmentation fault      (core dumped) $STEAM_DEBUGGER "$STEAMROOT/$STEAMEXEPATH" "$@"
assert_20160511174550_1.dmp[15979]: Finished uploading minidump (out-of-process): success = yes
assert_20160511174550_1.dmp[15979]: response: CrashID=bp-e5238e1c-4493-43c4-9161-20bc02160511
assert_20160511174550_1.dmp[15979]: file ''/tmp/dumps/assert_20160511174550_1.dmp'', upload yes: ''CrashID=bp-e5238e1c-4493-43c4-9161-20bc02160511''

**Comment by Plagman** [comment]
The crash seems related to your sound system. Are you running pulseaudio on this machine?

**Comment by jhall39** [comment]
No, I'm not. It runs alsa. I've had to use a ld prefetch command to load a newer alsa driver than steam uses to get the sound working before. But I had to use a bash script to run that and this error happens when I run it direct

**Comment by jhall39** [comment]
Okay, I'm going to eat some crow. I didn't think I had pulseaudio, only alsa. Apparently I do have pulseaudio and it is the source of all my pain. Seems like for some reason the module-alsa-source decided to stop working on me. It claimed the hw interface wasn't valid. Because of this, pulseaudio refused to start. I had to comment that out from my /etc/pulse/default.pa file and then edit the settings with pavucontrol instead. Suddenly it works. Even the sound works the first time. I used to have a lot of issues with the sound coming out of the onboard audio instead of the HDMI audio. Oddly, this problem didn't affect audio anywhere else on my system.
